src/game_interpreter.cpp

```diff
@@ -615,7 +615,10 @@ lcf::rpg::MoveCommand Game_Interpreter::DecodeMove(lcf::DBArray<int32_t>::const_
 bool Game_Interpreter::ExecuteCommand() {
```

Comment:
You can remove the no argument ``()`` version of this function from map and battle as they have the same code. And in game_interpreter.h you can delete the virtual as the function is not supposed to be overridden by sub classes

You said in chat this was needed because of a stackoverflow but I don't see why. Was probably a different issue.